components/Carbon.js

```diff
@@ -118,13 +137,19 @@ class Carbon extends PureComponent {
             onBeforeChange={(editor, meta, code) => this.codeUpdated(code)}
             value={this.props.children}
             options={options}
+            onUpdate={this.codeMirrorUpdate}
           />
           {config.watermark && <Watermark />}
           <div id="container-bg">
             <div className="white eliminateOnRender" />
             <div className="alpha eliminateOnRender" />
             <div className="bg" />
           </div>
+          <style>
```

Comment:
You can just put this style in the other `style` tag

Reply:
Do you mean the `style` tag that begins with `#container`? I tried, the style will not be applied

Reply:
That is just because your classes are created outside of react. You can just add this class and then dynamically apply styles within the template string something like:
```js
this.props.config.lineNumberSeperator  ? `border-right: ${"1px solid white"};` : ''
this.props.config.lineNumberSeperator  ? `padding-right: 10px important;` : ''
```

and then remove the `codeMirrorUpdate` altogether.